recipes/vs2008_runtime/bld.bat

```diff
@@ -0,0 +1,26 @@
+for /F "usebackq tokens=3*" %%A in (`REG QUERY "HKEY_LOCAL_MACHINE\Software\Microsoft\DevDiv\VC\Servicing\9.0\IDE" /v UpdateVersion`) do (
```

Comment:
Do you know how to query the runtime library version number from the registry or CMake for VS 2008, @isuruf? I'm pretty sure what I have here is wrong. Though without a Windows machine at hand, it's proving difficult to find out.

Reply:
Pushed an untested fix. Let's see if it works

Reply:
Thanks @isuruf. Nice fix!